Formula/w/whalebrew.rb

```diff
@@ -2,8 +2,8 @@ class Whalebrew < Formula
   desc "Homebrew, but with Docker images"
   homepage "https://github.com/whalebrew/whalebrew"
   url "https://github.com/whalebrew/whalebrew.git",
```

Comment:
Also we can update to just use the tarball instead of a git clone:

```suggestion
  url "https://github.com/whalebrew/whalebrew/archive/refs/tags/0.5.0.tar.gz"
  sha256 "2abea4171dbdca429b6476cffdfe7c94ce27028dc5d96e0f3a9a4fdeab77c4fb"
```

Reply:
Thank you for pointing it out, pushing my changes using release taball.